_guides/deploying-with-ksonnet.md

```diff
@@ -0,0 +1,519 @@
+---
+title: Deploying with KSonnet
+collection: How-to Guides
```

Comment:
Remove this line. The collection is defined by the folder `_guides`

Reply:
I did remove it. It didn't solve the problem. 
The original issue description according to your requirements -- http://prntscr.com/mki5aa

Reply:
Here is the solution:

---
title: Deploying with KSonnet
layout: article
section: Developing Components
category: component
order: 1
since: 20190119
---

Reply:
@al3xive no, this is not a solution. 

* `section` is wrong
* `category` needs to be removed